ftl-build/alpine/Dockerfile

```diff
@@ -82,6 +84,22 @@ RUN curl -sSL https://ftl.pi-hole.net/libraries/mbedtls-${mbedtlsversion}.tar.gz
     && cd .. \
     && rm -r mbedtls-${mbedtlsversion}
 
+# Build static libunwind (we cannot use the Alpine package because it depends on
+# xz and possibly other libraries)
```

Comment:
Is there a reason we can't add it to the `apk` list? I'm okay with using the supplied packages and `apk` should automatically pull in any dependencies.